trio_websocket/_impl.py

```diff
@@ -18,13 +18,16 @@
 
 from .version import __version__
 
+
+CONN_TIMEOUT = 30 # default connect & disconnect timeout, in seconds
```

Comment:
I'd opt for something longer, like 60 or 120.  People should be overriding this according to their use case.  The default should just be lest than infinite, and not small enough to be triggered by crappy software systems.